storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11208,6 +11209,85 @@ static void show_myrocks_vars(THD *thd, SHOW_VAR *var, char *buff) {
   var->value = reinterpret_cast<char *>(&myrocks_status_variables);
 }
 
+static ulonglong
+io_stall_prop_value(const std::map<std::string, std::string> &props,
+                    std::string &&key) {
```

Comment:
I think this can just be const std::string &key.

Reply:
Not sure if it possible as I pass a string directly to the method call, like 
io_stall_stats.level0_slowdown += io_stall_prop_value(props, "level0_slowdown");

LMK if I miss something.

Reply:
I tested this before when I saw your call with arguments like "level0_slowdown" and it looks to work with const std::string &key. Not clear to me if there is preference for one or the other.

Reply:
I've tested it again and it turned out my original test missed 'const'. If I use it then it works indeed. I will change it according to your comment.

Reply:
I've updated summary to highlight what rocksdb changes were done relevant to this PR.